### ISSUE TITLE
Warning: "Plugin CitizensCMD tried to register permission"

### ISSUE THREAD

**Issue by EvilG-MC** [HexedHero/CitizensCMD#10]
**Describe the bug**
A warning in the console related with the plugin permission.

**To Reproduce**
1. Download CitizensCMD v2.6.9 and Citizens
2. Put both plugins in your server.
3. Start the server.
4. See the error

**Expected behavior**
I'd like to think that well, don't throw out the warning.

**Screenshots**
![image](https://user-images.githubusercontent.com/71156616/235052193-90649951-b752-4fcc-adcc-e1a8799788cc.png)

**Versions:**
 - Citizens version: v2.0.31-SNAPSHOT (build 3060)
 - Spigot version: Paper Spigot 1.8.8
 - CitizensCMD version: v2.6.9
 - Java version: 11

**Comment by HexedHero** [comment]
Did you use /reload or a PlugMan reload?

**Comment by EvilG-MC** [comment]
> Did you use /reload or a PlugMan reload?

No, I also thought the same thing, but out of pure curiosity I opened a test server on server.pro I put citizens and citizenscmd and it happened anyway

**Comment by HexedHero** [comment]
I have no idea what the cause of this is.

**Comment by lc-company** [comment]
same issue here

**Comment by lc-company** [comment]
try delete permissions in plugin.yml? @HexedHero

**Comment by Runkang10** [comment]
Same thing on Paper plugin. I don't know if it's a problem with CommandAPI (In my case) or shading something that shouldn't be shaded.

**Comment by Runkang10** [comment]
@HexedHero To fix that you need to remove permission from `plugin.yml` or `paper-plugin.yml`, or just manually check if the user has the current permission to execute that command by removing `.withPermission(...)`.

**Comment by Runkang10** [comment]
@HexedHero ?
